src/boosting/gbdt_model_text.cpp

```diff
@@ -14,7 +14,7 @@
 
 namespace LightGBM {
 
-const std::string kModelVersion = "v2";
+const std::string kModelVersion = "v3";
```

Comment:
Is new model format backward compatible with current v2?

Reply:
Yeah, it can load the v2 model file